test/Bart/Git/CommitTest.php

```diff
@@ -30,6 +30,23 @@ private function stubGitRootForMessage($message)
 		});
 	}
 
+	private function stubGitRootFileList($getFileList)
+	{
+		$getFileList = "commit 31ac1584186257ce6aa14d9cefd78a4b2f89c90e
+Author: Nishad Singh <nsingh@box.com>
+Date:   Mon Aug 10 14:09:36 2015 -0700
+
+conf_override/features.conf
+conf_override/services.conf
+$getFileList";
+
+		$this->gitRoot = $this->shmock('\Bart\Git\GitRoot', function($root) use ($getFileList) {
+			$resultStub = new StubbedCommandResult([$getFileList], 0);
```

Comment:
The first parameter to `StubbedCommandResult()` should be an array of the individual lines of the output.